images/creds.proto

```diff
@@ -16,13 +16,14 @@ message creds_entry {
 	repeated uint32	cap_prm	= 10;
 	repeated uint32	cap_eff	= 11;
 	repeated uint32	cap_bnd	= 12;
+	repeated uint32 cap_amb = 13;
```

Comment:
You cannot change protobuf IDs. That breaks backwards compatibility. Always add additional fields at the end.

Reply:
done